content/docs/04-clusters/01-public-cloud/03-azure.md

```diff
@@ -41,136 +41,24 @@ The following prerequisites must be met before deploying a workload cluster in A
 
 3. You should have an Infrastructure cluster profile created in Palette for Azure cloud.
 
-# Creating an Azure Cloud Account
+# Section Highlights

```

Comment:
I'm not opposed to having this section, but if we do, we should do the same for the rest. Today, App Profile does not have a heading, we list the resources below.